content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -15,53 +16,45 @@ code to the rescue! We can now provision an entire EKS cluster with a
 CLI single gesture, thanks to [the `@pulumi/eks`
 package](https://github.com/pulumi/pulumi-eks). Let's see how.
 
-![EasyEKS](https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=1800&name=EasyEKS.png){width="1800"
-sizes="(max-width: 1800px) 100vw, 1800px"
-srcset="https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=900&name=EasyEKS.png 900w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=1800&name=EasyEKS.png 1800w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=2700&name=EasyEKS.png 2700w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=3600&name=EasyEKS.png 3600w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=4500&name=EasyEKS.png 4500w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=5400&name=EasyEKS.png 5400w"}
+![Pulumi making EKS Easy](./easy-eks.png)
 
-To get started, download the free and open source [Pulumi
-SDK](https://pulumi.io/quickstart/install.html). The SDK includes
+To get started, download the free and open source
+[Pulumi SDK]({{< ref "/docs/quickstart" >}}). The SDK includes
 the CLI we'll be using below, and requires AWS credentials to access
 your AWS account and provision resources. If you already have the AWS
 CLI installed and configured, you're already all set to go. Otherwise,
-[check out the docs here](https://pulumi.io/quickstart/aws/setup.html)
+[check out the docs here]({{< ref "/docs/quickstart/aws" >}})
```

Comment:
This should probably link to the page on actually getting the AWS account setup, which is now `/docs/reference/clouds/aws/setup.md`

Reply:
Done